app/src/main/java/com/nextcloud/client/preferences/AppPreferencesImpl.java

```diff
@@ -92,12 +95,12 @@ public final class AppPreferencesImpl implements AppPreferences {
     private static final String PREF__PIN_BRUTE_FORCE_COUNT = "pin_brute_force_count";
     private static final String PREF__UID_PID = "uid_pid";
 
-    private static final String PREF__CALENDAR_AUTOMATIC_BACKUP = "calendar_automatic_backup";
```

Comment:
I will remove this upstream, so will be soon gone.

Reply:
I have removed from this PR to avoid any conflicts when you will removed from your end.